instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/InetSocketAddressNetClientAttributesGetter.java

```diff
@@ -29,9 +29,6 @@ public final String peerName(REQUEST request, @Nullable RESPONSE response) {
     if (address == null) {
       return null;
     }
-    if (address.getAddress() != null) {
-      return address.getAddress().getHostName();
-    }
     return address.getHostString();
```

Comment:
I believe `getHostString` may return ip address. I hope spec is ok with this, if it isn't I guess we could compare it with `getHostAddress` and return null in case they match.

Reply:
We're already doing that: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/NetClientAttributesExtractor.java#L51-L53